doc/cephfs/mount-using-kernel-driver.rst

```diff
@@ -50,51 +56,52 @@ Synopsis
+  mount -t ceph {device-string}={path-to-mounted} {mount-point} -o {key-value-args} {other-args}
 
 Mounting CephFS
 ===============
 On Ceph clusters, CephX is enabled by default. Use ``mount`` command to
 mount CephFS with the kernel driver::
 
-    mkdir /mnt/mycephfs
-    mount -t ceph :/ /mnt/mycephfs -o name=foo
+  mkdir /mnt/mycephfs
+  mount -t ceph <name>@<fsid>.<fs_name>=/ /mnt/mycephfs
 
-The key-value argument right after option ``-o`` is CephX credential;
-``name`` is the username of the CephX user we are using to mount CephFS. The
-default value for ``name`` is ``guest``.
+``name`` is the username of the CephX user we are using to mount CephFS.
+``fsid`` is the FSID of the ceph cluster which can be found using
+``ceph fsid`` command. ``fs_name`` is the file system to mount. The kernel
+driver requires MON's socket and the secret key for the CephX user, e.g.::
 
-The kernel driver also requires MON's socket and the secret key for the CephX
-user. In case of the above command, ``mount.ceph`` helper figures out these
-details automatically by finding and reading Ceph conf file and keyring. In
-case you don't have these files on the host where you're running mount
-command, you can pass these details yourself too::
+  mount -t ceph cephuser@b3acfc0d-575f-41d3-9c91-0e7ed3dbb3fa.cephfs=/ -o mon_addr=192.168.0.1:6789,secret=AQATSKdNGBnwLhAAnNDKnH65FmVKpXZJVasUeQ==
 
-    mount -t ceph 192.168.0.1:6789,192.168.0.2:6789:/ /mnt/mycephfs -o name=foo,secret=AQATSKdNGBnwLhAAnNDKnH65FmVKpXZJVasUeQ==
+When using the mount helper, monitor hosts and FSID are optional. ``mount.ceph``
+helper figures out these details automatically by finding and reading ceph conf
+file, .e.g::
 
-Passing a single MON socket in above command works too. A potential problem
-with the command above is that the secret key is left in your shell's command
-history. To prevent that you can copy the secret key inside a file and pass
-the file by using the option ``secretfile`` instead of ``secret``::
+  mount -t ceph cephuser@.cephfs=/ -o secret=AQATSKdNGBnwLhAAnNDKnH65FmVKpXZJVasUeQ==
 
-    mount -t ceph :/ /mnt/mycephfs -o name=foo,secretfile=/etc/ceph/foo.secret
+.. note:: Note that the dot (``.``) still needs to be a part of the device string.
 
-Ensure the permissions on the secret key file are appropriate (preferably,
-``600``).
+A potential problem with the above command is that the secret key is left in your
+shell's command history. To prevent that you can copy the secret key inside a file
+and pass the file by using the option ``secretfile`` instead of ``secret``::
 
-In case CephX is disabled, you can omit ``-o`` and the list of key-value
-arguments that follow it::
+  mount -t ceph cephuser@.cephfs=/ /mnt/mycephfs -o secretfile=/etc/ceph/cephuser.secret
 
-    mount -t ceph :/ /mnt/mycephfs
+Ensure the permissions on the secret key file are appropriate (preferably, ``600``).
```

Comment:
``0600``

The '0' prefix makes it clear that it's octal.

Reply:
ACK